.gitignore

```diff
@@ -1,4 +1,33 @@
-node_modules
-dist
-.vscode/launch.json
 test/configfiles
+
+# compiled output
+/dist
+/node_modules
+
+# Logs
+logs
+*.log
+npm-debug.log*
+pnpm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+
+# OS
+.DS_Store
```

Comment:
os-specific, this one should be in the developer .gitignore, not in the project

Reply:
Ok, i will remove all of these too :
```# IDEs and editors
/.idea
.project
.classpath
.c9/
*.launch
.settings/
*.sublime-workspace

# IDE - VSCode
.vscode/*
```